src/lightning/pytorch/strategies/xla_fsdp.py

```diff
@@ -414,8 +433,6 @@ def save_checkpoint(
                 consolidate_sharded_model_checkpoints(ckpt_prefix, ckpt_suffix, str(save_path))
                 # remove the shards directory
                 self.checkpoint_io.remove_checkpoint(path)
-                # mv the consolidated checkpoint where the user would expect it
-                get_filesystem(save_path).mv(str(save_path), str(path))
```

Comment:
Why did you remove this?

Reply:
Whoops, I can add this back. I was having issues with one of the tests but forgot to add back this snippet.

Reply:
Done